.github/workflows/code-check.yml

```diff
@@ -45,7 +45,7 @@ jobs:
           cache: true
 
       - name: Run Flutter Format
-        run: dart format --set-exit-if-changed .
+        run: dart format .
```

Comment:
pls lets keep `--set-exit-if-changed` because we want to formatter to tell when there are any format issues in the code